recipe/meta.yaml

```diff
@@ -23,18 +23,22 @@ requirements:
     - bluesky-darkframes
     - caproto
     - nslsii >=0.1.0
+    - inflection
     - pexpect
+    - pillow
     - pyepics >=3.4.2
     - pyolog >=4.4.0
     - pyserial
     - python-confluent-kafka  # [not win]
     - redis-py
+    - slack-sdk
```

Comment:
We only need to add this package. The other 3 are provided implicitly as dependencies of the `analysis` feedstock:
- [`inflection`](https://github.com/nsls-ii-forge/analysis-feedstock/blob/de531d3075e6d016db2c11c06513293c0078e25a/recipe/meta.yaml#L44)
- [`pillow`](https://github.com/nsls-ii-forge/analysis-feedstock/blob/de531d3075e6d016db2c11c06513293c0078e25a/recipe/meta.yaml#L73)
- [`xraylarch`](https://github.com/nsls-ii-forge/analysis-feedstock/blob/de531d3075e6d016db2c11c06513293c0078e25a/recipe/meta.yaml#L113)